src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskType;
+import duke.task.ToDo;
+
+import static duke.task.TaskType.DEADLINE;
+import static duke.task.TaskType.EVENT;
+import static duke.task.TaskType.TODO;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        DukeList dList = new DukeList();
+        String[] greet = {getLogo(), "Hello! I'm Duke", "What can I do for you?"};
+
+        printOut(greet);
+        while (true) {
+            String cmd = scan.nextLine();
+            String[] arguments = cmd.split(" ");
+            try {
+                if (arguments[0].equals("bye")) {
```

Comment:
For magic literals like "bye", "list", "done" etc. Since they will not be changing, should they be made into a constant?